src/houston/public/styles/index.css

```diff
@@ -27,42 +27,29 @@ footer ul li a:focus i.fa {
 }
 
 /*******
-* Hero *
+* Cards *
 *******/
 
-section.hero {
-    background-color: #4d4267;
-    background-image: linear-gradient(to right bottom, #5d4e75, #4d4267);
-    color: #fff;
-    text-align: center;
-    text-shadow: 0 1px 2px rgba(0, 0, 0, 0.3);
-    padding: 120px 0 64px;
-    max-width: 100%;
-    position: relative;
-}
-
-section.hero > div {
-    position: relative;
-}
-
-section.hero .button.suggested-action {
-    max-width: 100%;
-}
-
-section.hero .small-label a {
-    color: inherit;
-    font-weight: bold;
-}
-
-section.hero .small-label a:hover {
-    text-decoration: underline;
+.card {
+    background-color: #fff;
+    border-radius: 3px;
+    box-shadow:
+        0 0 0 1px rgba(0,0,0,0.15),
+        0 8px 10px 1px rgba(0,0,0,0.07),
+        0 3px 14px 2px rgba(0,0,0,0.06),
+        0 5px 5px -3px rgba(0,0,0,0.2);
+    margin: 16px;
+    padding: 24px;
+    transition: all 200ms cubic-bezier(0.4, 0.0, 0.2, 1);
 }
 
-sup {
-    font-family: 'Open Sans';
-    font-size: 16px;
-    font-weight: 600;
-    margin-left: 12px;
+.card:hover {
```

Comment:
I feel like non-linked cards should not move when hovered, or perhaps should not be cards at all. The transform implies something will happen when clicked, and in the case of the Stripe and GitHub cards, they're not links.